infra/feast-operator/test/api/featurestore_types_test.go

```diff
@@ -111,6 +146,77 @@ func registryWithRelativePathForEphemeral(featureStore *feastdevv1alpha1.Feature
 	}
 	return copy
 }
+func registryWithS3BucketForPvc(featureStore *feastdevv1alpha1.FeatureStore) *feastdevv1alpha1.FeatureStore {
+	copy := featureStore.DeepCopy()
+	copy.Spec.Services = &feastdevv1alpha1.FeatureStoreServices{
+		Registry: &feastdevv1alpha1.Registry{
+			Local: &feastdevv1alpha1.LocalRegistryConfig{
+				Persistence: &feastdevv1alpha1.RegistryPersistence{
+					FilePersistence: &feastdevv1alpha1.RegistryFilePersistence{
+						Path: "s3://bucket/registry.db",
+						PvcConfig: &feastdevv1alpha1.PvcConfig{
+							Create:    &feastdevv1alpha1.PvcCreate{},
+							MountPath: "/data/registry",
+						},
+					},
+				},
+			},
+		},
+	}
+	return copy
+}
+func registryWithGsBucketForPvc(featureStore *feastdevv1alpha1.FeatureStore) *feastdevv1alpha1.FeatureStore {
+	copy := featureStore.DeepCopy()
+	copy.Spec.Services = &feastdevv1alpha1.FeatureStoreServices{
+		Registry: &feastdevv1alpha1.Registry{
+			Local: &feastdevv1alpha1.LocalRegistryConfig{
+				Persistence: &feastdevv1alpha1.RegistryPersistence{
+					FilePersistence: &feastdevv1alpha1.RegistryFilePersistence{
+						Path: "gs://bucket/registry.db",
```

Comment:
above method `registryWithS3BucketForPvc` and this method seems to be the same except difference in the path. can we just pass that as a method param?